public/pages/HistoricalDetectorList/utils/constants.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import {
+  EuiLink,
+  EuiText,
+  EuiToolTip,
+  EuiHealth,
+  EuiBasicTableColumn,
+} from '@elastic/eui';
+//@ts-ignore
+import moment from 'moment';
+import React from 'react';
+import { get } from 'lodash';
+import { Detector } from '../../../models/interfaces';
+import { PLUGIN_NAME } from '../../../utils/constants';
+import { DETECTOR_STATE } from '../../../../server/utils/constants';
+import { stateToColorMap } from '../../utils/constants';
+
+const DEFAULT_EMPTY_DATA = '-';
+const columnStyle = {
+  overflow: 'visible',
+  whiteSpace: 'normal',
+  wordBreak: 'break-word',
+} as React.CSSProperties;
+
+const renderTime = (time: number) => {
+  const momentTime = moment(time);
+  if (time && momentTime.isValid())
+    return momentTime.format('MM/DD/YYYY h:mm A');
+  return DEFAULT_EMPTY_DATA;
+};
+
+const renderState = (state: DETECTOR_STATE) => {
```

Comment:
why not use existing [method](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/14168a737f361203d0b2434006b04d55c65c0dfd/public/pages/DetectorsList/utils/tableUtils.tsx#L49)? we can just export existing one.

Reply:
same comment for above methods/constant

Reply:
good catch, will refactor that. Originally did that because I was going to split up the historical v. realtime states.